doc/source/user_guide/cookbook.rst

```diff
@@ -1488,3 +1488,31 @@ of the data values:
        {"height": [60, 70], "weight": [100, 140, 180], "sex": ["Male", "Female"]}
    )
    df
+
+Constant series
+---------------
+
+To assess if a series has a constant value, we can check if ``series.nunique() == 1``.
+However, a more performant approach, that does not count all unique values first, is:
+
+.. ipython:: python
+
+   v = s.values
+   is_constant = v.shape[0] > 0 and (s[0] == s).all()
```

Comment:
I would describe the empty Series as being constant; much in the same way an empty function is: https://math.stackexchange.com/questions/1949584/is-the-empty-function-constant

Reply:
I agree (and it means that the commonly used `nunique() == 1` should be `nunique() <= 1`). 
Let me update the snippet